frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureStrategyMenu.tsx

```diff
@@ -154,6 +168,27 @@ export const FeatureStrategyMenu = ({
 
     return (
         <StyledStrategyMenu onClick={(event) => event.stopPropagation()}>
+            {displayReleasePlanButton() ? (
```

Comment:
I see these components are using the <ConditionallyRender> but there's a whole [ADR](https://docs.getunleash.io/contributing/ADRs/front-end/jsx-conditionals) to stop doing that so... I've chosen to use ternaries instead

Reply:
I don't think that'll matter, actually. Both of those come from hooks so it should cause a re-render anyway. I don't have strong feelings either way so I'm happy to change it to a const